bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs/diff/MergePlan.scala

```diff
@@ -75,24 +102,23 @@ case class MergePlan(update: Tree[MergeNode], delete: Set[Missing]) {
       }
     }
 
-    val nodeMap = p.nodeMap
-
     // Pair up MergeNodes with their corresponding ISPNode, creating any missing
     // ISPNodes as necessary.
-    val mergeTree = update.map { mn =>
-      nodeMap.get(mn.key).fold(create(mn)) { _.right }.map {n => (mn, n) }
-    }.sequenceU
+    val mergeTree: VcsAction[Tree[(MergeNode, ISPNode)]] = Task.delay {
+      val nodeMap = p.nodeMap
 
+      update.map { mn =>
+        nodeMap.get(mn.key).fold(create(mn)) { _.right }.map {n => (mn, n) }
+      }.sequenceU
```

Comment:
Two other ways to be cool:
- `fa.map(a => (x, a))` can be written `fa.strengthL(x)`
- `fa.map(f).sequenceU` can be written `fa.traverseU(f)`